api/rs/slint/Cargo.toml

```diff
@@ -78,6 +78,12 @@ unsafe-single-threaded = ["i-slint-core/unsafe-single-threaded"]
 ## APIs to support screen readers and other assistive technologies.
 accessibility = ["i-slint-backend-selector/accessibility"]
 
+## Enable integration with [raw-window-handle](raw_window_handle_06) version 0.6. This provides a
+## [`Window::window_handle()`] function that returns a struct that implements
+## [HasWindowHandle](raw_window_handle_06::HasWindowHandle) and
+## [HasDisplayHandle](raw_window_handle_06::HasDisplayHandle) implementation.
+raw-window-handle-06 = ["dep:raw-window-handle-06", "i-slint-backend-selector/raw-window-handle-06"]
```

Comment:
should we add the feature for docs.rs as well? (down in the metadata)

Reply:
Ohh yess. Thanks!